pkg/operator/app/pipeline.go

```diff
@@ -100,11 +100,9 @@ func ConfigurePipelineParts(ctx context.Context, p *PipelineParts) error {
 		return err
 	}
 
-	if p.Deps.ToolInjectionCheckout.Satisfied() {
-		p.Pipeline.SetWorkspace(tektonv1beta1.PipelineWorkspaceDeclaration{
-			Name: ToolsWorkspaceName,
-		})
-	}
+	p.Pipeline.SetWorkspace(tektonv1beta1.PipelineWorkspaceDeclaration{
```

Comment:
I'm almost positive that you can remove all this workspace-specific stuff too, including anything that might mount it into the containers.

Reply:
Yeah we can, but I was going to reuse it. (although we don't specifically need to...)
But the code is currently written/functional using the Tekton workspace for the tool mounts.